OrbitClientModel/CaptureData.cpp

```diff
@@ -112,6 +112,22 @@ const std::string& CaptureData::GetFunctionNameByAddress(uint64_t absolute_addre
   return function_name;
 }
 
+// Find the start address of the function this address falls inside. Use the Function returned by
+// FindFunctionByAddress, and when this fails (e.g., the module containing the function has not
+// been loaded) use (for now) the LinuxAddressInfo that is collected for every address in a
+// callstack.
+uint64_t CaptureData::GetFunctionBaseAddressByAddress(uint64_t absolute_address) const {
+  const FunctionInfo* function = FindFunctionByAddress(absolute_address, false);
+  if (function != nullptr) {
+    return GetAbsoluteAddress(*function);
+  }
+  const LinuxAddressInfo* address_info = GetAddressInfo(absolute_address);
+  if (address_info != nullptr) {
+    return absolute_address - address_info->offset_in_function();
+  }
```

Comment:
I wonder if we should translate address infos to functions after capture is done so that this function can be simplified.

Reply:
IIRC @antonrohr and I had the discussion at one point in time, but I don't remember what our arguments where.
I agree that the distinction of LinuxAddressData and FunctionInfo is not intuitive. This was the main motivation to keep this local to CaptureData where ever possible, and use the functions provided here. 
So, I'd argue, that this change is a step in this direction.

Also note, that as soon as we turn on frame-pointer-based unwinding, LinuxAddressInfo will be completely gone anyways.